src/indexer/services/monitoringService/index.ts

```diff
@@ -0,0 +1,42 @@
+import { CronJob } from 'cron'
+import TransferRepository from '../../repository/transfer';
+import { logger } from '../../../utils/logger';
+import { DateTime } from 'luxon';
+import { NotificationSender } from './notificationSender';
+
+export async function checkTransferStatus(transferRepository: TransferRepository): Promise<void>{
+
+    const notificationSender = new NotificationSender("region")
+
+    const transfers = await transferRepository.findTransfersByStatus("pending")
+    console.log("--- Cronjob is running ---")
```

Comment:
I think we have a proper logger to replace this

Reply:
Also, best to put this at the begging of the function and name it `--- Checking pending transfers ---`